openserver/openserver.py

```diff
@@ -71,12 +71,19 @@ def DoGet(self, Gv):
         if not self.status == 'Connected':
             self.connect()
         try:
-            DoGet = self.server.GetValue(Gv)
+            value = self.server.GetValue(Gv)
             AppName = self.GetAppName(Gv)
             Err = self.server.GetLastError(AppName)
             if Err > 0:
                 print(self.server.GetLastErrorMessage(AppName))
-            return DoGet
+            if value.isdigit():  # Checking if integer
+                value = int(value)
+            else:
+                try:
+                    value = float(value)  # Checking if float
```

Comment:
Do we need to check for both int and floats? Wouldn't a float(value) check suffice?

Reply:
I see now that we should.